### DISCUSSION TITLE
Highlight current line

### DISCUSSION THREAD

**Discussion by lisiur** [helix-editor/helix#6117] [Q&A] [answered]
Can helix highlight current line?

**Comment by CptPotato** [comment, marked as answer]
Yes, there's an [editor setting called `cursorline`](https://docs.helix-editor.com/configuration.html?highlight=cursorline#editor-section).

**Reply by lisiur** [reply]
Thanks for your reply. I'm using base16_transparent theme but it seems not work?

**Reply by CptPotato** [reply]
Looks like `base16_transparent` doesn't specify a cursorline color, so it's not showing up with this specific theme. I'm not sure if this is intended.

You could set the color yourself if you want to stick with `base16_transparent` by creating a custom theme and adding the `ui.cursorline` key like this:

```toml
inherits = "base16_transparent"

"ui.cursorline" = "#abcdef"
```

You can add the custom theme to your themes directory (`~/.config/helix/themes/`).

**Reply by lisiur** [reply]
Thanks!
